[2023-10-18T17:52:35.055+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: games_details.test_transform scheduled__2023-10-17T00:00:00+00:00 [queued]>
[2023-10-18T17:52:35.063+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: games_details.test_transform scheduled__2023-10-17T00:00:00+00:00 [queued]>
[2023-10-18T17:52:35.063+0000] {taskinstance.py:1359} INFO - Starting attempt 4 of 4
[2023-10-18T17:52:35.074+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonVirtualenvOperator): test_transform> on 2023-10-17 00:00:00+00:00
[2023-10-18T17:52:35.080+0000] {standard_task_runner.py:57} INFO - Started process 1304 to run task
[2023-10-18T17:52:35.085+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'games_details', 'test_transform', 'scheduled__2023-10-17T00:00:00+00:00', '--job-id', '548', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpmzn87lqd']
[2023-10-18T17:52:35.087+0000] {standard_task_runner.py:85} INFO - Job 548: Subtask test_transform
[2023-10-18T17:52:35.128+0000] {task_command.py:415} INFO - Running <TaskInstance: games_details.test_transform scheduled__2023-10-17T00:00:00+00:00 [running]> on host 3c47de805a7a
[2023-10-18T17:52:35.199+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='games_details' AIRFLOW_CTX_TASK_ID='test_transform' AIRFLOW_CTX_EXECUTION_DATE='2023-10-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-17T00:00:00+00:00'
[2023-10-18T17:52:35.202+0000] {process_utils.py:182} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venvp5oen6ll
[2023-10-18T17:52:35.215+0000] {process_utils.py:186} INFO - Output:
[2023-10-18T17:52:37.144+0000] {process_utils.py:190} INFO - created virtual environment CPython3.8.18.final.0-64 in 1385ms
[2023-10-18T17:52:37.145+0000] {process_utils.py:190} INFO -   creator CPython3Posix(dest=/tmp/venvp5oen6ll, clear=False, no_vcs_ignore=False, global=False)
[2023-10-18T17:52:37.145+0000] {process_utils.py:190} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/tmp/tmp3pi4s0sx)
[2023-10-18T17:52:37.145+0000] {process_utils.py:190} INFO -     added seed packages: pip==23.2, setuptools==68.0.0, wheel==0.40.0
[2023-10-18T17:52:37.145+0000] {process_utils.py:190} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2023-10-18T17:52:37.167+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venvp5oen6ll/bin/pip install -r /tmp/venvp5oen6ll/requirements.txt
[2023-10-18T17:52:37.179+0000] {process_utils.py:186} INFO - Output:
[2023-10-18T17:52:38.464+0000] {process_utils.py:190} INFO - WARNING: The directory '/home/***/.cache/pip' or its parent directory is not owned or is not writable by the current user. The cache has been disabled. Check the permissions and owner of that directory. If executing pip with sudo, you should use sudo's -H flag.
[2023-10-18T17:52:38.573+0000] {process_utils.py:190} INFO - Looking in indexes: https://pypi.cloud.soda.io
[2023-10-18T17:52:39.641+0000] {process_utils.py:190} INFO - Collecting soda-core-bigquery (from -r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:39.641+0000] {process_utils.py:190} INFO -   Obtaining dependency information for soda-core-bigquery from https://files.pythonhosted.org/packages/8b/70/9e1eb02fd7f3814314c6f4c89e4e3c5f040135d9b97ec4504bedefa063c3/soda_core_bigquery-3.0.51-py3-none-any.whl.metadata
[2023-10-18T17:52:39.921+0000] {process_utils.py:190} INFO -   Downloading soda_core_bigquery-3.0.51-py3-none-any.whl.metadata (149 bytes)
[2023-10-18T17:52:40.441+0000] {process_utils.py:190} INFO - Collecting soda-core==3.0.51 (from soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:40.442+0000] {process_utils.py:190} INFO -   Obtaining dependency information for soda-core==3.0.51 from https://files.pythonhosted.org/packages/25/9c/c15271d048ecf6d6f2b638fa3e62812c54297a3d985fee13794171bcab44/soda_core-3.0.51-py3-none-any.whl.metadata
[2023-10-18T17:52:40.458+0000] {process_utils.py:190} INFO -   Downloading soda_core-3.0.51-py3-none-any.whl.metadata (1.0 kB)
[2023-10-18T17:52:41.024+0000] {process_utils.py:190} INFO - Collecting google-cloud-bigquery<4.0,>=2.25.0 (from soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:41.025+0000] {process_utils.py:190} INFO -   Obtaining dependency information for google-cloud-bigquery<4.0,>=2.25.0 from https://files.pythonhosted.org/packages/22/03/a5d04741507156b88e9101e91d97bd395d043e9de280b94b1d00cb0f4e03/google_cloud_bigquery-3.12.0-py2.py3-none-any.whl.metadata
[2023-10-18T17:52:41.045+0000] {process_utils.py:190} INFO -   Downloading google_cloud_bigquery-3.12.0-py2.py3-none-any.whl.metadata (8.5 kB)
[2023-10-18T17:52:41.614+0000] {process_utils.py:190} INFO - Collecting markupsafe<=2.1.2,>=2.0.1 (from soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:41.627+0000] {process_utils.py:190} INFO -   Downloading MarkupSafe-2.1.2-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (25 kB)
[2023-10-18T17:52:42.178+0000] {process_utils.py:190} INFO - Collecting Jinja2<4.0,>=2.11 (from soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:42.197+0000] {process_utils.py:190} INFO -   Downloading Jinja2-3.1.2-py3-none-any.whl (133 kB)
[2023-10-18T17:52:42.212+0000] {process_utils.py:190} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 133.1/133.1 kB 10.1 MB/s eta 0:00:00
[2023-10-18T17:52:42.717+0000] {process_utils.py:190} INFO - Collecting click~=8.0 (from soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:42.718+0000] {process_utils.py:190} INFO -   Obtaining dependency information for click~=8.0 from https://files.pythonhosted.org/packages/00/2e/d53fa4befbf2cfa713304affc7ca780ce4fc1fd8710527771b58311a3229/click-8.1.7-py3-none-any.whl.metadata
[2023-10-18T17:52:42.730+0000] {process_utils.py:190} INFO -   Downloading click-8.1.7-py3-none-any.whl.metadata (3.0 kB)
[2023-10-18T17:52:43.562+0000] {process_utils.py:190} INFO - Collecting ruamel.yaml<0.18.0,>=0.17.0 (from soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:43.563+0000] {process_utils.py:190} INFO -   Obtaining dependency information for ruamel.yaml<0.18.0,>=0.17.0 from https://files.pythonhosted.org/packages/79/a9/ae2f346226fcec5beb172ae3cab6d47ae0baa986f1ed1bdf08d96b469b27/ruamel.yaml-0.17.35-py3-none-any.whl.metadata
[2023-10-18T17:52:43.575+0000] {process_utils.py:190} INFO -   Downloading ruamel.yaml-0.17.35-py3-none-any.whl.metadata (18 kB)
[2023-10-18T17:52:44.100+0000] {process_utils.py:190} INFO - Collecting requests~=2.28 (from soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:44.101+0000] {process_utils.py:190} INFO -   Obtaining dependency information for requests~=2.28 from https://files.pythonhosted.org/packages/70/8e/0e2d847013cb52cd35b38c009bb167a1a26b2ce6cd6965bf26b47bc0bf44/requests-2.31.0-py3-none-any.whl.metadata
[2023-10-18T17:52:44.115+0000] {process_utils.py:190} INFO -   Downloading requests-2.31.0-py3-none-any.whl.metadata (4.6 kB)
[2023-10-18T17:52:44.614+0000] {process_utils.py:190} INFO - Collecting antlr4-python3-runtime~=4.11.1 (from soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:44.631+0000] {process_utils.py:190} INFO -   Downloading antlr4_python3_runtime-4.11.1-py3-none-any.whl (144 kB)
[2023-10-18T17:52:44.644+0000] {process_utils.py:190} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 144.2/144.2 kB 12.2 MB/s eta 0:00:00
[2023-10-18T17:52:45.155+0000] {process_utils.py:190} INFO - Collecting opentelemetry-api~=1.16.0 (from soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:45.173+0000] {process_utils.py:190} INFO -   Downloading opentelemetry_api-1.16.0-py3-none-any.whl (57 kB)
[2023-10-18T17:52:45.179+0000] {process_utils.py:190} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 57.3/57.3 kB 14.2 MB/s eta 0:00:00
[2023-10-18T17:52:45.685+0000] {process_utils.py:190} INFO - Collecting opentelemetry-exporter-otlp-proto-http~=1.16.0 (from soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:45.703+0000] {process_utils.py:190} INFO -   Downloading opentelemetry_exporter_otlp_proto_http-1.16.0-py3-none-any.whl (21 kB)
[2023-10-18T17:52:46.207+0000] {process_utils.py:190} INFO - Collecting sqlparse~=0.4 (from soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:46.220+0000] {process_utils.py:190} INFO -   Downloading sqlparse-0.4.4-py3-none-any.whl (41 kB)
[2023-10-18T17:52:46.223+0000] {process_utils.py:190} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 41.2/41.2 kB 144.5 MB/s eta 0:00:00
[2023-10-18T17:52:46.728+0000] {process_utils.py:190} INFO - Collecting inflect~=6.0 (from soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:46.729+0000] {process_utils.py:190} INFO -   Obtaining dependency information for inflect~=6.0 from https://files.pythonhosted.org/packages/6a/e1/79603e4ae91b38a91dfd855f92321afb65e7f3d0192ac831d00bc9c45cca/inflect-6.2.0-py3-none-any.whl.metadata
[2023-10-18T17:52:46.744+0000] {process_utils.py:190} INFO -   Downloading inflect-6.2.0-py3-none-any.whl.metadata (21 kB)
[2023-10-18T17:52:48.176+0000] {process_utils.py:190} INFO - Collecting grpcio<2.0dev,>=1.47.0 (from google-cloud-bigquery<4.0,>=2.25.0->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:48.176+0000] {process_utils.py:190} INFO -   Obtaining dependency information for grpcio<2.0dev,>=1.47.0 from https://files.pythonhosted.org/packages/d6/a1/c5ca3fb951edb7bab892e61e24c72fc7359ae48916cf7352cde9d712867d/grpcio-1.59.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
[2023-10-18T17:52:48.187+0000] {process_utils.py:190} INFO -   Downloading grpcio-1.59.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (4.0 kB)
[2023-10-18T17:52:48.742+0000] {process_utils.py:190} INFO - Collecting google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5 (from google-cloud-bigquery<4.0,>=2.25.0->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:48.743+0000] {process_utils.py:190} INFO -   Obtaining dependency information for google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5 from https://files.pythonhosted.org/packages/4d/ce/4fd62ea66b3508debc795e475336ce915929765870f0ad52328426ba016e/google_api_core-2.12.0-py3-none-any.whl.metadata
[2023-10-18T17:52:48.753+0000] {process_utils.py:190} INFO -   Downloading google_api_core-2.12.0-py3-none-any.whl.metadata (2.7 kB)
[2023-10-18T17:52:49.262+0000] {process_utils.py:190} INFO - Collecting proto-plus<2.0.0dev,>=1.15.0 (from google-cloud-bigquery<4.0,>=2.25.0->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:49.263+0000] {process_utils.py:190} INFO -   Obtaining dependency information for proto-plus<2.0.0dev,>=1.15.0 from https://files.pythonhosted.org/packages/36/5b/e02636d221917d6fa2a61289b3f16002eb4c93d51c0191ac8e896d527182/proto_plus-1.22.3-py3-none-any.whl.metadata
[2023-10-18T17:52:49.278+0000] {process_utils.py:190} INFO -   Downloading proto_plus-1.22.3-py3-none-any.whl.metadata (2.2 kB)
[2023-10-18T17:52:49.790+0000] {process_utils.py:190} INFO - Collecting google-cloud-core<3.0.0dev,>=1.6.0 (from google-cloud-bigquery<4.0,>=2.25.0->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:49.790+0000] {process_utils.py:190} INFO -   Obtaining dependency information for google-cloud-core<3.0.0dev,>=1.6.0 from https://files.pythonhosted.org/packages/a2/40/02045f776fdb6e44194f34b6375a26ce8a61bd9bd03cd8930ed91cf51a62/google_cloud_core-2.3.3-py2.py3-none-any.whl.metadata
[2023-10-18T17:52:49.802+0000] {process_utils.py:190} INFO -   Downloading google_cloud_core-2.3.3-py2.py3-none-any.whl.metadata (2.5 kB)
[2023-10-18T17:52:50.312+0000] {process_utils.py:190} INFO - Collecting google-resumable-media<3.0dev,>=0.6.0 (from google-cloud-bigquery<4.0,>=2.25.0->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:50.313+0000] {process_utils.py:190} INFO -   Obtaining dependency information for google-resumable-media<3.0dev,>=0.6.0 from https://files.pythonhosted.org/packages/c7/4f/b8e5e22406e5aeafa46df8799939d5eeee52f18eeed339675167fac6603a/google_resumable_media-2.6.0-py2.py3-none-any.whl.metadata
[2023-10-18T17:52:50.326+0000] {process_utils.py:190} INFO -   Downloading google_resumable_media-2.6.0-py2.py3-none-any.whl.metadata (2.1 kB)
[2023-10-18T17:52:50.835+0000] {process_utils.py:190} INFO - Collecting packaging>=20.0.0 (from google-cloud-bigquery<4.0,>=2.25.0->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:50.835+0000] {process_utils.py:190} INFO -   Obtaining dependency information for packaging>=20.0.0 from https://files.pythonhosted.org/packages/ec/1a/610693ac4ee14fcdf2d9bf3c493370e4f2ef7ae2e19217d7a237ff42367d/packaging-23.2-py3-none-any.whl.metadata
[2023-10-18T17:52:50.846+0000] {process_utils.py:190} INFO -   Downloading packaging-23.2-py3-none-any.whl.metadata (3.2 kB)
[2023-10-18T17:52:51.679+0000] {process_utils.py:190} INFO - Collecting protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5 (from google-cloud-bigquery<4.0,>=2.25.0->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:51.679+0000] {process_utils.py:190} INFO -   Obtaining dependency information for protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5 from https://files.pythonhosted.org/packages/c8/2c/03046cac73f46bfe98fc846ef629cf4f84c2f59258216aa2cc0d22bfca8f/protobuf-4.24.4-cp37-abi3-manylinux2014_x86_64.whl.metadata
[2023-10-18T17:52:51.690+0000] {process_utils.py:190} INFO -   Downloading protobuf-4.24.4-cp37-abi3-manylinux2014_x86_64.whl.metadata (540 bytes)
[2023-10-18T17:52:52.184+0000] {process_utils.py:190} INFO - Collecting python-dateutil<3.0dev,>=2.7.2 (from google-cloud-bigquery<4.0,>=2.25.0->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:52.199+0000] {process_utils.py:190} INFO -   Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
[2023-10-18T17:52:52.221+0000] {process_utils.py:190} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 247.7/247.7 kB 11.9 MB/s eta 0:00:00
[2023-10-18T17:52:52.759+0000] {process_utils.py:190} INFO - Collecting googleapis-common-protos<2.0.dev0,>=1.56.2 (from google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-cloud-bigquery<4.0,>=2.25.0->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:52.759+0000] {process_utils.py:190} INFO -   Obtaining dependency information for googleapis-common-protos<2.0.dev0,>=1.56.2 from https://files.pythonhosted.org/packages/21/49/12996dc0238e017504dceea1d121a48bd49fb3f4416f40d59fc3e924b4f3/googleapis_common_protos-1.61.0-py2.py3-none-any.whl.metadata
[2023-10-18T17:52:52.775+0000] {process_utils.py:190} INFO -   Downloading googleapis_common_protos-1.61.0-py2.py3-none-any.whl.metadata (1.5 kB)
[2023-10-18T17:52:53.362+0000] {process_utils.py:190} INFO - Collecting google-auth<3.0.dev0,>=2.14.1 (from google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-cloud-bigquery<4.0,>=2.25.0->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:53.362+0000] {process_utils.py:190} INFO -   Obtaining dependency information for google-auth<3.0.dev0,>=2.14.1 from https://files.pythonhosted.org/packages/39/7c/2e4fa55a99f83ef9ef229ac5d59c44ceb90e2d0145711590c0fa39669f32/google_auth-2.23.3-py2.py3-none-any.whl.metadata
[2023-10-18T17:52:53.378+0000] {process_utils.py:190} INFO -   Downloading google_auth-2.23.3-py2.py3-none-any.whl.metadata (4.2 kB)
[2023-10-18T17:52:53.928+0000] {process_utils.py:190} INFO - Collecting grpcio-status<2.0.dev0,>=1.33.2 (from google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-cloud-bigquery<4.0,>=2.25.0->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:53.928+0000] {process_utils.py:190} INFO -   Obtaining dependency information for grpcio-status<2.0.dev0,>=1.33.2 from https://files.pythonhosted.org/packages/55/ce/e6d0382610240439ced22fe2183bcc387946bf80e5e0f17f5b5250978ff3/grpcio_status-1.59.0-py3-none-any.whl.metadata
[2023-10-18T17:52:53.939+0000] {process_utils.py:190} INFO -   Downloading grpcio_status-1.59.0-py3-none-any.whl.metadata (1.3 kB)
[2023-10-18T17:52:54.510+0000] {process_utils.py:190} INFO - Collecting google-crc32c<2.0dev,>=1.0 (from google-resumable-media<3.0dev,>=0.6.0->google-cloud-bigquery<4.0,>=2.25.0->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:54.521+0000] {process_utils.py:190} INFO -   Downloading google_crc32c-1.5.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (32 kB)
[2023-10-18T17:52:55.277+0000] {process_utils.py:190} INFO - Collecting pydantic>=1.9.1 (from inflect~=6.0->soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:55.277+0000] {process_utils.py:190} INFO -   Obtaining dependency information for pydantic>=1.9.1 from https://files.pythonhosted.org/packages/73/66/0a72c9fcde42e5650c8d8d5c5c1873b9a3893018020c77ca8eb62708b923/pydantic-2.4.2-py3-none-any.whl.metadata
[2023-10-18T17:52:55.294+0000] {process_utils.py:190} INFO -   Downloading pydantic-2.4.2-py3-none-any.whl.metadata (158 kB)
[2023-10-18T17:52:55.308+0000] {process_utils.py:190} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 158.6/158.6 kB 12.4 MB/s eta 0:00:00
[2023-10-18T17:52:55.806+0000] {process_utils.py:190} INFO - Collecting typing-extensions (from inflect~=6.0->soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:55.807+0000] {process_utils.py:190} INFO -   Obtaining dependency information for typing-extensions from https://files.pythonhosted.org/packages/24/21/7d397a4b7934ff4028987914ac1044d3b7d52712f30e2ac7a2ae5bc86dd0/typing_extensions-4.8.0-py3-none-any.whl.metadata
[2023-10-18T17:52:55.817+0000] {process_utils.py:190} INFO -   Downloading typing_extensions-4.8.0-py3-none-any.whl.metadata (3.0 kB)
[2023-10-18T17:52:56.354+0000] {process_utils.py:190} INFO - Collecting deprecated>=1.2.6 (from opentelemetry-api~=1.16.0->soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:56.354+0000] {process_utils.py:190} INFO -   Obtaining dependency information for deprecated>=1.2.6 from https://files.pythonhosted.org/packages/20/8d/778b7d51b981a96554f29136cd59ca7880bf58094338085bcf2a979a0e6a/Deprecated-1.2.14-py2.py3-none-any.whl.metadata
[2023-10-18T17:52:56.369+0000] {process_utils.py:190} INFO -   Downloading Deprecated-1.2.14-py2.py3-none-any.whl.metadata (5.4 kB)
[2023-10-18T17:52:56.370+0000] {process_utils.py:190} INFO - Requirement already satisfied: setuptools>=16.0 in /tmp/venvp5oen6ll/lib/python3.8/site-packages (from opentelemetry-api~=1.16.0->soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2)) (68.0.0)
[2023-10-18T17:52:56.883+0000] {process_utils.py:190} INFO - Collecting backoff<3.0.0,>=1.10.0 (from opentelemetry-exporter-otlp-proto-http~=1.16.0->soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:56.899+0000] {process_utils.py:190} INFO -   Downloading backoff-2.2.1-py3-none-any.whl (15 kB)
[2023-10-18T17:52:57.403+0000] {process_utils.py:190} INFO - Collecting opentelemetry-proto==1.16.0 (from opentelemetry-exporter-otlp-proto-http~=1.16.0->soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:57.414+0000] {process_utils.py:190} INFO -   Downloading opentelemetry_proto-1.16.0-py3-none-any.whl (52 kB)
[2023-10-18T17:52:57.419+0000] {process_utils.py:190} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 52.6/52.6 kB 17.2 MB/s eta 0:00:00
[2023-10-18T17:52:57.924+0000] {process_utils.py:190} INFO - Collecting opentelemetry-sdk~=1.16.0 (from opentelemetry-exporter-otlp-proto-http~=1.16.0->soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:57.939+0000] {process_utils.py:190} INFO -   Downloading opentelemetry_sdk-1.16.0-py3-none-any.whl (94 kB)
[2023-10-18T17:52:57.947+0000] {process_utils.py:190} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 94.6/94.6 kB 12.9 MB/s eta 0:00:00
[2023-10-18T17:52:58.533+0000] {process_utils.py:190} INFO - Collecting six>=1.5 (from python-dateutil<3.0dev,>=2.7.2->google-cloud-bigquery<4.0,>=2.25.0->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:58.547+0000] {process_utils.py:190} INFO -   Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
[2023-10-18T17:52:59.151+0000] {process_utils.py:190} INFO - Collecting charset-normalizer<4,>=2 (from requests~=2.28->soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:59.151+0000] {process_utils.py:190} INFO -   Obtaining dependency information for charset-normalizer<4,>=2 from https://files.pythonhosted.org/packages/1e/c8/fd52271326c052f95f47ef718b018aa2bc3fd097d9bac44d7d48894c6130/charset_normalizer-3.3.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
[2023-10-18T17:52:59.166+0000] {process_utils.py:190} INFO -   Downloading charset_normalizer-3.3.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (32 kB)
[2023-10-18T17:52:59.653+0000] {process_utils.py:190} INFO - Collecting idna<4,>=2.5 (from requests~=2.28->soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:52:59.665+0000] {process_utils.py:190} INFO -   Downloading idna-3.4-py3-none-any.whl (61 kB)
[2023-10-18T17:52:59.671+0000] {process_utils.py:190} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.5/61.5 kB 14.2 MB/s eta 0:00:00
[2023-10-18T17:53:00.213+0000] {process_utils.py:190} INFO - Collecting urllib3<3,>=1.21.1 (from requests~=2.28->soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:53:00.213+0000] {process_utils.py:190} INFO -   Obtaining dependency information for urllib3<3,>=1.21.1 from https://files.pythonhosted.org/packages/d2/b2/b157855192a68541a91ba7b2bbcb91f1b4faa51f8bae38d8005c034be524/urllib3-2.0.7-py3-none-any.whl.metadata
[2023-10-18T17:53:00.223+0000] {process_utils.py:190} INFO -   Downloading urllib3-2.0.7-py3-none-any.whl.metadata (6.6 kB)
[2023-10-18T17:53:00.725+0000] {process_utils.py:190} INFO - Collecting certifi>=2017.4.17 (from requests~=2.28->soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:53:00.726+0000] {process_utils.py:190} INFO -   Obtaining dependency information for certifi>=2017.4.17 from https://files.pythonhosted.org/packages/4c/dd/2234eab22353ffc7d94e8d13177aaa050113286e93e7b40eae01fbf7c3d9/certifi-2023.7.22-py3-none-any.whl.metadata
[2023-10-18T17:53:00.738+0000] {process_utils.py:190} INFO -   Downloading certifi-2023.7.22-py3-none-any.whl.metadata (2.2 kB)
[2023-10-18T17:53:01.286+0000] {process_utils.py:190} INFO - Collecting ruamel.yaml.clib>=0.2.7 (from ruamel.yaml<0.18.0,>=0.17.0->soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:53:01.286+0000] {process_utils.py:190} INFO -   Obtaining dependency information for ruamel.yaml.clib>=0.2.7 from https://files.pythonhosted.org/packages/22/fa/b2a8fd49c92693e9b9b6b11eef4c2a8aedaca2b521ab3e020aa4778efc23/ruamel.yaml.clib-0.2.8-cp38-cp38-manylinux_2_5_x86_64.manylinux1_x86_64.whl.metadata
[2023-10-18T17:53:01.297+0000] {process_utils.py:190} INFO -   Downloading ruamel.yaml.clib-0.2.8-cp38-cp38-manylinux_2_5_x86_64.manylinux1_x86_64.whl.metadata (2.2 kB)
[2023-10-18T17:53:01.968+0000] {process_utils.py:190} INFO - Collecting wrapt<2,>=1.10 (from deprecated>=1.2.6->opentelemetry-api~=1.16.0->soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:53:01.984+0000] {process_utils.py:190} INFO -   Downloading wrapt-1.15.0-cp38-cp38-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (81 kB)
[2023-10-18T17:53:01.988+0000] {process_utils.py:190} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 81.5/81.5 kB 233.6 MB/s eta 0:00:00
[2023-10-18T17:53:02.512+0000] {process_utils.py:190} INFO - Collecting cachetools<6.0,>=2.0.0 (from google-auth<3.0.dev0,>=2.14.1->google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-cloud-bigquery<4.0,>=2.25.0->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:53:02.513+0000] {process_utils.py:190} INFO -   Obtaining dependency information for cachetools<6.0,>=2.0.0 from https://files.pythonhosted.org/packages/a9/c9/c8a7710f2cedcb1db9224fdd4d8307c9e48cbddc46c18b515fefc0f1abbe/cachetools-5.3.1-py3-none-any.whl.metadata
[2023-10-18T17:53:02.526+0000] {process_utils.py:190} INFO -   Downloading cachetools-5.3.1-py3-none-any.whl.metadata (5.2 kB)
[2023-10-18T17:53:03.054+0000] {process_utils.py:190} INFO - Collecting pyasn1-modules>=0.2.1 (from google-auth<3.0.dev0,>=2.14.1->google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-cloud-bigquery<4.0,>=2.25.0->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:53:03.071+0000] {process_utils.py:190} INFO -   Downloading pyasn1_modules-0.3.0-py2.py3-none-any.whl (181 kB)
[2023-10-18T17:53:03.083+0000] {process_utils.py:190} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 181.3/181.3 kB 30.1 MB/s eta 0:00:00
[2023-10-18T17:53:03.576+0000] {process_utils.py:190} INFO - Collecting rsa<5,>=3.1.4 (from google-auth<3.0.dev0,>=2.14.1->google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-cloud-bigquery<4.0,>=2.25.0->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:53:03.590+0000] {process_utils.py:190} INFO -   Downloading rsa-4.9-py3-none-any.whl (34 kB)
[2023-10-18T17:53:04.207+0000] {process_utils.py:190} INFO - Collecting opentelemetry-semantic-conventions==0.37b0 (from opentelemetry-sdk~=1.16.0->opentelemetry-exporter-otlp-proto-http~=1.16.0->soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:53:04.218+0000] {process_utils.py:190} INFO -   Downloading opentelemetry_semantic_conventions-0.37b0-py3-none-any.whl (26 kB)
[2023-10-18T17:53:04.732+0000] {process_utils.py:190} INFO - Collecting annotated-types>=0.4.0 (from pydantic>=1.9.1->inflect~=6.0->soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:53:04.733+0000] {process_utils.py:190} INFO -   Obtaining dependency information for annotated-types>=0.4.0 from https://files.pythonhosted.org/packages/28/78/d31230046e58c207284c6b2c4e8d96e6d3cb4e52354721b944d3e1ee4aa5/annotated_types-0.6.0-py3-none-any.whl.metadata
[2023-10-18T17:53:04.746+0000] {process_utils.py:190} INFO -   Downloading annotated_types-0.6.0-py3-none-any.whl.metadata (12 kB)
[2023-10-18T17:53:06.272+0000] {process_utils.py:190} INFO - Collecting pydantic-core==2.10.1 (from pydantic>=1.9.1->inflect~=6.0->soda-core==3.0.51->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:53:06.272+0000] {process_utils.py:190} INFO -   Obtaining dependency information for pydantic-core==2.10.1 from https://files.pythonhosted.org/packages/da/82/4139ea7aebec9e22be42d8a5342df5022b7b6b6deb10126dc3ff80b4d6fd/pydantic_core-2.10.1-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
[2023-10-18T17:53:06.288+0000] {process_utils.py:190} INFO -   Downloading pydantic_core-2.10.1-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (6.5 kB)
[2023-10-18T17:53:06.961+0000] {process_utils.py:190} INFO - Collecting pyasn1<0.6.0,>=0.4.6 (from pyasn1-modules>=0.2.1->google-auth<3.0.dev0,>=2.14.1->google-api-core[grpc]!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-cloud-bigquery<4.0,>=2.25.0->soda-core-bigquery->-r /tmp/venvp5oen6ll/requirements.txt (line 2))
[2023-10-18T17:53:06.975+0000] {process_utils.py:190} INFO -   Downloading pyasn1-0.5.0-py2.py3-none-any.whl (83 kB)
[2023-10-18T17:53:06.982+0000] {process_utils.py:190} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 83.9/83.9 kB 13.2 MB/s eta 0:00:00
[2023-10-18T17:53:07.051+0000] {process_utils.py:190} INFO - Downloading soda_core_bigquery-3.0.51-py3-none-any.whl (3.7 kB)
[2023-10-18T17:53:07.067+0000] {process_utils.py:190} INFO - Downloading soda_core-3.0.51-py3-none-any.whl (178 kB)
[2023-10-18T17:53:07.083+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 178.9/178.9 kB 12.3 MB/s eta 0:00:00
[2023-10-18T17:53:07.098+0000] {process_utils.py:190} INFO - Downloading google_cloud_bigquery-3.12.0-py2.py3-none-any.whl (220 kB)
[2023-10-18T17:53:07.117+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 220.4/220.4 kB 12.2 MB/s eta 0:00:00
[2023-10-18T17:53:07.132+0000] {process_utils.py:190} INFO - Downloading click-8.1.7-py3-none-any.whl (97 kB)
[2023-10-18T17:53:07.141+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 97.9/97.9 kB 13.0 MB/s eta 0:00:00
[2023-10-18T17:53:07.160+0000] {process_utils.py:190} INFO - Downloading google_cloud_core-2.3.3-py2.py3-none-any.whl (29 kB)
[2023-10-18T17:53:07.179+0000] {process_utils.py:190} INFO - Downloading google_resumable_media-2.6.0-py2.py3-none-any.whl (80 kB)
[2023-10-18T17:53:07.183+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 80.3/80.3 kB 240.1 MB/s eta 0:00:00
[2023-10-18T17:53:07.201+0000] {process_utils.py:190} INFO - Downloading grpcio-1.59.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (5.3 MB)
[2023-10-18T17:53:07.651+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 5.3/5.3 MB 11.9 MB/s eta 0:00:00
[2023-10-18T17:53:07.664+0000] {process_utils.py:190} INFO - Downloading inflect-6.2.0-py3-none-any.whl (35 kB)
[2023-10-18T17:53:07.677+0000] {process_utils.py:190} INFO - Downloading packaging-23.2-py3-none-any.whl (53 kB)
[2023-10-18T17:53:07.682+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 53.0/53.0 kB 15.2 MB/s eta 0:00:00
[2023-10-18T17:53:07.708+0000] {process_utils.py:190} INFO - Downloading proto_plus-1.22.3-py3-none-any.whl (48 kB)
[2023-10-18T17:53:07.712+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 48.1/48.1 kB 178.2 MB/s eta 0:00:00
[2023-10-18T17:53:07.729+0000] {process_utils.py:190} INFO - Downloading protobuf-4.24.4-cp37-abi3-manylinux2014_x86_64.whl (311 kB)
[2023-10-18T17:53:07.752+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 311.6/311.6 kB 63.4 MB/s eta 0:00:00
[2023-10-18T17:53:07.765+0000] {process_utils.py:190} INFO - Downloading requests-2.31.0-py3-none-any.whl (62 kB)
[2023-10-18T17:53:07.771+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 62.6/62.6 kB 30.6 MB/s eta 0:00:00
[2023-10-18T17:53:07.790+0000] {process_utils.py:190} INFO - Downloading ruamel.yaml-0.17.35-py3-none-any.whl (112 kB)
[2023-10-18T17:53:07.797+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 112.9/112.9 kB 222.0 MB/s eta 0:00:00
[2023-10-18T17:53:07.819+0000] {process_utils.py:190} INFO - Downloading certifi-2023.7.22-py3-none-any.whl (158 kB)
[2023-10-18T17:53:07.829+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 158.3/158.3 kB 46.0 MB/s eta 0:00:00
[2023-10-18T17:53:07.849+0000] {process_utils.py:190} INFO - Downloading charset_normalizer-3.3.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (137 kB)
[2023-10-18T17:53:07.857+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 137.9/137.9 kB 52.2 MB/s eta 0:00:00
[2023-10-18T17:53:07.873+0000] {process_utils.py:190} INFO - Downloading Deprecated-1.2.14-py2.py3-none-any.whl (9.6 kB)
[2023-10-18T17:53:07.893+0000] {process_utils.py:190} INFO - Downloading google_api_core-2.12.0-py3-none-any.whl (121 kB)
[2023-10-18T17:53:07.903+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 121.4/121.4 kB 12.6 MB/s eta 0:00:00
[2023-10-18T17:53:07.920+0000] {process_utils.py:190} INFO - Downloading google_auth-2.23.3-py2.py3-none-any.whl (182 kB)
[2023-10-18T17:53:07.936+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 182.3/182.3 kB 12.3 MB/s eta 0:00:00
[2023-10-18T17:53:07.953+0000] {process_utils.py:190} INFO - Downloading googleapis_common_protos-1.61.0-py2.py3-none-any.whl (230 kB)
[2023-10-18T17:53:07.972+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 230.9/230.9 kB 12.7 MB/s eta 0:00:00
[2023-10-18T17:53:07.985+0000] {process_utils.py:190} INFO - Downloading grpcio_status-1.59.0-py3-none-any.whl (14 kB)
[2023-10-18T17:53:08.002+0000] {process_utils.py:190} INFO - Downloading pydantic-2.4.2-py3-none-any.whl (395 kB)
[2023-10-18T17:53:08.037+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 395.8/395.8 kB 11.9 MB/s eta 0:00:00
[2023-10-18T17:53:08.052+0000] {process_utils.py:190} INFO - Downloading pydantic_core-2.10.1-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (2.1 MB)
[2023-10-18T17:53:08.228+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.1/2.1 MB 11.8 MB/s eta 0:00:00
[2023-10-18T17:53:08.244+0000] {process_utils.py:190} INFO - Downloading ruamel.yaml.clib-0.2.8-cp38-cp38-manylinux_2_5_x86_64.manylinux1_x86_64.whl (596 kB)
[2023-10-18T17:53:08.296+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 596.0/596.0 kB 11.9 MB/s eta 0:00:00
[2023-10-18T17:53:08.311+0000] {process_utils.py:190} INFO - Downloading typing_extensions-4.8.0-py3-none-any.whl (31 kB)
[2023-10-18T17:53:08.325+0000] {process_utils.py:190} INFO - Downloading urllib3-2.0.7-py3-none-any.whl (124 kB)
[2023-10-18T17:53:08.336+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 124.2/124.2 kB 13.1 MB/s eta 0:00:00
[2023-10-18T17:53:08.354+0000] {process_utils.py:190} INFO - Downloading annotated_types-0.6.0-py3-none-any.whl (12 kB)
[2023-10-18T17:53:08.370+0000] {process_utils.py:190} INFO - Downloading cachetools-5.3.1-py3-none-any.whl (9.3 kB)
[2023-10-18T17:53:08.637+0000] {process_utils.py:190} INFO - Installing collected packages: antlr4-python3-runtime, wrapt, urllib3, typing-extensions, sqlparse, six, ruamel.yaml.clib, pyasn1, protobuf, packaging, opentelemetry-semantic-conventions, markupsafe, idna, grpcio, google-crc32c, click, charset-normalizer, certifi, cachetools, backoff, ruamel.yaml, rsa, requests, python-dateutil, pydantic-core, pyasn1-modules, proto-plus, opentelemetry-proto, Jinja2, googleapis-common-protos, google-resumable-media, deprecated, annotated-types, pydantic, opentelemetry-api, grpcio-status, google-auth, opentelemetry-sdk, inflect, google-api-core, opentelemetry-exporter-otlp-proto-http, google-cloud-core, soda-core, google-cloud-bigquery, soda-core-bigquery
[2023-10-18T17:53:10.733+0000] {process_utils.py:190} INFO - Successfully installed Jinja2-3.1.2 annotated-types-0.6.0 antlr4-python3-runtime-4.11.1 backoff-2.2.1 cachetools-5.3.1 certifi-2023.7.22 charset-normalizer-3.3.0 click-8.1.7 deprecated-1.2.14 google-api-core-2.12.0 google-auth-2.23.3 google-cloud-bigquery-3.12.0 google-cloud-core-2.3.3 google-crc32c-1.5.0 google-resumable-media-2.6.0 googleapis-common-protos-1.61.0 grpcio-1.59.0 grpcio-status-1.59.0 idna-3.4 inflect-6.2.0 markupsafe-2.1.2 opentelemetry-api-1.16.0 opentelemetry-exporter-otlp-proto-http-1.16.0 opentelemetry-proto-1.16.0 opentelemetry-sdk-1.16.0 opentelemetry-semantic-conventions-0.37b0 packaging-23.2 proto-plus-1.22.3 protobuf-4.24.4 pyasn1-0.5.0 pyasn1-modules-0.3.0 pydantic-2.4.2 pydantic-core-2.10.1 python-dateutil-2.8.2 requests-2.31.0 rsa-4.9 ruamel.yaml-0.17.35 ruamel.yaml.clib-0.2.8 six-1.16.0 soda-core-3.0.51 soda-core-bigquery-3.0.51 sqlparse-0.4.4 typing-extensions-4.8.0 urllib3-2.0.7 wrapt-1.15.0
[2023-10-18T17:53:10.881+0000] {process_utils.py:190} INFO - 
[2023-10-18T17:53:10.881+0000] {process_utils.py:190} INFO - [notice] A new release of pip is available: 23.2 -> 23.3
[2023-10-18T17:53:10.881+0000] {process_utils.py:190} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2023-10-18T17:53:11.186+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venvp5oen6ll/bin/python /tmp/venvp5oen6ll/script.py /tmp/venvp5oen6ll/script.in /tmp/venvp5oen6ll/script.out /tmp/venvp5oen6ll/string_args.txt /tmp/venvp5oen6ll/termination.log
[2023-10-18T17:53:11.197+0000] {process_utils.py:186} INFO - Output:
[2023-10-18T17:53:11.955+0000] {process_utils.py:190} INFO - [17:53:11] Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_price_overview.yml
[2023-10-18T17:53:11.955+0000] {process_utils.py:190} INFO -   | 'NoneType' object is not iterable
[2023-10-18T17:53:11.956+0000] {process_utils.py:190} INFO -   | Stacktrace:
[2023-10-18T17:53:11.956+0000] {process_utils.py:190} INFO -   | Traceback (most recent call last):
[2023-10-18T17:53:11.956+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 304, in add_sodacl_yaml_file
[2023-10-18T17:53:11.956+0000] {process_utils.py:190} INFO -   |     self._parse_sodacl_yaml_str(sodacl_yaml_str=sodacl_yaml_str, file_path=file_path)
[2023-10-18T17:53:11.957+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 335, in _parse_sodacl_yaml_str
[2023-10-18T17:53:11.957+0000] {process_utils.py:190} INFO -   |     sodacl_parser.parse_sodacl_yaml_str(sodacl_yaml_str)
[2023-10-18T17:53:11.957+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 115, in parse_sodacl_yaml_str
[2023-10-18T17:53:11.957+0000] {process_utils.py:190} INFO -   |     self.__parse_headers(sodacl_dict)
[2023-10-18T17:53:11.958+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 151, in __parse_headers
[2023-10-18T17:53:11.958+0000] {process_utils.py:190} INFO -   |     self.__parse_table_checks_section(
[2023-10-18T17:53:11.958+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 215, in __parse_table_checks_section
[2023-10-18T17:53:11.959+0000] {process_utils.py:190} INFO -   |     check_str, check_configurations = self.__parse_check_configuration(check_list_element)
[2023-10-18T17:53:11.959+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 1759, in __parse_check_configuration
[2023-10-18T17:53:11.959+0000] {process_utils.py:190} INFO -   |     ignored_config_keys = [k for k in check_configurations if k != check_str]
[2023-10-18T17:53:11.959+0000] {process_utils.py:190} INFO -   | TypeError: 'NoneType' object is not iterable
[2023-10-18T17:53:11.960+0000] {process_utils.py:190} INFO - 
[2023-10-18T17:53:11.969+0000] {process_utils.py:190} INFO - [17:53:11] Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/fct_game_pricing.yml
[2023-10-18T17:53:11.969+0000] {process_utils.py:190} INFO -   | 'NoneType' object is not iterable
[2023-10-18T17:53:11.970+0000] {process_utils.py:190} INFO -   | Stacktrace:
[2023-10-18T17:53:11.970+0000] {process_utils.py:190} INFO -   | Traceback (most recent call last):
[2023-10-18T17:53:11.970+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 304, in add_sodacl_yaml_file
[2023-10-18T17:53:11.970+0000] {process_utils.py:190} INFO -   |     self._parse_sodacl_yaml_str(sodacl_yaml_str=sodacl_yaml_str, file_path=file_path)
[2023-10-18T17:53:11.971+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 335, in _parse_sodacl_yaml_str
[2023-10-18T17:53:11.971+0000] {process_utils.py:190} INFO -   |     sodacl_parser.parse_sodacl_yaml_str(sodacl_yaml_str)
[2023-10-18T17:53:11.971+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 115, in parse_sodacl_yaml_str
[2023-10-18T17:53:11.971+0000] {process_utils.py:190} INFO -   |     self.__parse_headers(sodacl_dict)
[2023-10-18T17:53:11.971+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 151, in __parse_headers
[2023-10-18T17:53:11.971+0000] {process_utils.py:190} INFO -   |     self.__parse_table_checks_section(
[2023-10-18T17:53:11.972+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 215, in __parse_table_checks_section
[2023-10-18T17:53:11.972+0000] {process_utils.py:190} INFO -   |     check_str, check_configurations = self.__parse_check_configuration(check_list_element)
[2023-10-18T17:53:11.972+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 1759, in __parse_check_configuration
[2023-10-18T17:53:11.972+0000] {process_utils.py:190} INFO -   |     ignored_config_keys = [k for k in check_configurations if k != check_str]
[2023-10-18T17:53:11.972+0000] {process_utils.py:190} INFO -   | TypeError: 'NoneType' object is not iterable
[2023-10-18T17:53:11.973+0000] {process_utils.py:190} INFO - 
[2023-10-18T17:53:11.974+0000] {process_utils.py:190} INFO - [17:53:11] Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_publishers.yml
[2023-10-18T17:53:11.974+0000] {process_utils.py:190} INFO -   | 'NoneType' object is not iterable
[2023-10-18T17:53:11.974+0000] {process_utils.py:190} INFO -   | Stacktrace:
[2023-10-18T17:53:11.975+0000] {process_utils.py:190} INFO -   | Traceback (most recent call last):
[2023-10-18T17:53:11.975+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 304, in add_sodacl_yaml_file
[2023-10-18T17:53:11.975+0000] {process_utils.py:190} INFO -   |     self._parse_sodacl_yaml_str(sodacl_yaml_str=sodacl_yaml_str, file_path=file_path)
[2023-10-18T17:53:11.975+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 335, in _parse_sodacl_yaml_str
[2023-10-18T17:53:11.975+0000] {process_utils.py:190} INFO -   |     sodacl_parser.parse_sodacl_yaml_str(sodacl_yaml_str)
[2023-10-18T17:53:11.976+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 115, in parse_sodacl_yaml_str
[2023-10-18T17:53:11.976+0000] {process_utils.py:190} INFO -   |     self.__parse_headers(sodacl_dict)
[2023-10-18T17:53:11.976+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 151, in __parse_headers
[2023-10-18T17:53:11.976+0000] {process_utils.py:190} INFO -   |     self.__parse_table_checks_section(
[2023-10-18T17:53:11.976+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 215, in __parse_table_checks_section
[2023-10-18T17:53:11.977+0000] {process_utils.py:190} INFO -   |     check_str, check_configurations = self.__parse_check_configuration(check_list_element)
[2023-10-18T17:53:11.977+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 1759, in __parse_check_configuration
[2023-10-18T17:53:11.977+0000] {process_utils.py:190} INFO -   |     ignored_config_keys = [k for k in check_configurations if k != check_str]
[2023-10-18T17:53:11.977+0000] {process_utils.py:190} INFO -   | TypeError: 'NoneType' object is not iterable
[2023-10-18T17:53:11.978+0000] {process_utils.py:190} INFO - 
[2023-10-18T17:53:11.982+0000] {process_utils.py:190} INFO - [17:53:11] Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_release.yml
[2023-10-18T17:53:11.983+0000] {process_utils.py:190} INFO -   | 'NoneType' object is not iterable
[2023-10-18T17:53:11.983+0000] {process_utils.py:190} INFO -   | Stacktrace:
[2023-10-18T17:53:11.983+0000] {process_utils.py:190} INFO -   | Traceback (most recent call last):
[2023-10-18T17:53:11.983+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 304, in add_sodacl_yaml_file
[2023-10-18T17:53:11.984+0000] {process_utils.py:190} INFO -   |     self._parse_sodacl_yaml_str(sodacl_yaml_str=sodacl_yaml_str, file_path=file_path)
[2023-10-18T17:53:11.984+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 335, in _parse_sodacl_yaml_str
[2023-10-18T17:53:11.984+0000] {process_utils.py:190} INFO -   |     sodacl_parser.parse_sodacl_yaml_str(sodacl_yaml_str)
[2023-10-18T17:53:11.984+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 115, in parse_sodacl_yaml_str
[2023-10-18T17:53:11.984+0000] {process_utils.py:190} INFO -   |     self.__parse_headers(sodacl_dict)
[2023-10-18T17:53:11.984+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 151, in __parse_headers
[2023-10-18T17:53:11.985+0000] {process_utils.py:190} INFO -   |     self.__parse_table_checks_section(
[2023-10-18T17:53:11.985+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 215, in __parse_table_checks_section
[2023-10-18T17:53:11.985+0000] {process_utils.py:190} INFO -   |     check_str, check_configurations = self.__parse_check_configuration(check_list_element)
[2023-10-18T17:53:11.985+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 1759, in __parse_check_configuration
[2023-10-18T17:53:11.985+0000] {process_utils.py:190} INFO -   |     ignored_config_keys = [k for k in check_configurations if k != check_str]
[2023-10-18T17:53:11.985+0000] {process_utils.py:190} INFO -   | TypeError: 'NoneType' object is not iterable
[2023-10-18T17:53:11.986+0000] {process_utils.py:190} INFO - 
[2023-10-18T17:53:11.987+0000] {process_utils.py:190} INFO - [17:53:11] Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_support_language.yml
[2023-10-18T17:53:11.987+0000] {process_utils.py:190} INFO -   | 'NoneType' object is not iterable
[2023-10-18T17:53:11.987+0000] {process_utils.py:190} INFO -   | Stacktrace:
[2023-10-18T17:53:11.987+0000] {process_utils.py:190} INFO -   | Traceback (most recent call last):
[2023-10-18T17:53:11.987+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 304, in add_sodacl_yaml_file
[2023-10-18T17:53:11.987+0000] {process_utils.py:190} INFO -   |     self._parse_sodacl_yaml_str(sodacl_yaml_str=sodacl_yaml_str, file_path=file_path)
[2023-10-18T17:53:11.987+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 335, in _parse_sodacl_yaml_str
[2023-10-18T17:53:11.988+0000] {process_utils.py:190} INFO -   |     sodacl_parser.parse_sodacl_yaml_str(sodacl_yaml_str)
[2023-10-18T17:53:11.988+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 115, in parse_sodacl_yaml_str
[2023-10-18T17:53:11.988+0000] {process_utils.py:190} INFO -   |     self.__parse_headers(sodacl_dict)
[2023-10-18T17:53:11.988+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 151, in __parse_headers
[2023-10-18T17:53:11.988+0000] {process_utils.py:190} INFO -   |     self.__parse_table_checks_section(
[2023-10-18T17:53:11.988+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 215, in __parse_table_checks_section
[2023-10-18T17:53:11.989+0000] {process_utils.py:190} INFO -   |     check_str, check_configurations = self.__parse_check_configuration(check_list_element)
[2023-10-18T17:53:11.989+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 1759, in __parse_check_configuration
[2023-10-18T17:53:11.989+0000] {process_utils.py:190} INFO -   |     ignored_config_keys = [k for k in check_configurations if k != check_str]
[2023-10-18T17:53:11.989+0000] {process_utils.py:190} INFO -   | TypeError: 'NoneType' object is not iterable
[2023-10-18T17:53:11.989+0000] {process_utils.py:190} INFO - 
[2023-10-18T17:53:11.993+0000] {process_utils.py:190} INFO - [17:53:11] Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_games.yml
[2023-10-18T17:53:11.993+0000] {process_utils.py:190} INFO -   | 'NoneType' object is not iterable
[2023-10-18T17:53:11.993+0000] {process_utils.py:190} INFO -   | Stacktrace:
[2023-10-18T17:53:11.994+0000] {process_utils.py:190} INFO -   | Traceback (most recent call last):
[2023-10-18T17:53:11.994+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 304, in add_sodacl_yaml_file
[2023-10-18T17:53:11.994+0000] {process_utils.py:190} INFO -   |     self._parse_sodacl_yaml_str(sodacl_yaml_str=sodacl_yaml_str, file_path=file_path)
[2023-10-18T17:53:11.994+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 335, in _parse_sodacl_yaml_str
[2023-10-18T17:53:11.994+0000] {process_utils.py:190} INFO -   |     sodacl_parser.parse_sodacl_yaml_str(sodacl_yaml_str)
[2023-10-18T17:53:11.995+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 115, in parse_sodacl_yaml_str
[2023-10-18T17:53:11.995+0000] {process_utils.py:190} INFO -   |     self.__parse_headers(sodacl_dict)
[2023-10-18T17:53:11.995+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 151, in __parse_headers
[2023-10-18T17:53:11.995+0000] {process_utils.py:190} INFO -   |     self.__parse_table_checks_section(
[2023-10-18T17:53:11.995+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 215, in __parse_table_checks_section
[2023-10-18T17:53:11.996+0000] {process_utils.py:190} INFO -   |     check_str, check_configurations = self.__parse_check_configuration(check_list_element)
[2023-10-18T17:53:11.996+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 1759, in __parse_check_configuration
[2023-10-18T17:53:11.996+0000] {process_utils.py:190} INFO -   |     ignored_config_keys = [k for k in check_configurations if k != check_str]
[2023-10-18T17:53:11.996+0000] {process_utils.py:190} INFO -   | TypeError: 'NoneType' object is not iterable
[2023-10-18T17:53:11.996+0000] {process_utils.py:190} INFO - 
[2023-10-18T17:53:11.997+0000] {process_utils.py:190} INFO - [17:53:11] Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_developers.yml
[2023-10-18T17:53:11.997+0000] {process_utils.py:190} INFO -   | 'NoneType' object is not iterable
[2023-10-18T17:53:11.997+0000] {process_utils.py:190} INFO -   | Stacktrace:
[2023-10-18T17:53:11.997+0000] {process_utils.py:190} INFO -   | Traceback (most recent call last):
[2023-10-18T17:53:11.997+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 304, in add_sodacl_yaml_file
[2023-10-18T17:53:11.998+0000] {process_utils.py:190} INFO -   |     self._parse_sodacl_yaml_str(sodacl_yaml_str=sodacl_yaml_str, file_path=file_path)
[2023-10-18T17:53:11.998+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 335, in _parse_sodacl_yaml_str
[2023-10-18T17:53:11.998+0000] {process_utils.py:190} INFO -   |     sodacl_parser.parse_sodacl_yaml_str(sodacl_yaml_str)
[2023-10-18T17:53:11.998+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 115, in parse_sodacl_yaml_str
[2023-10-18T17:53:11.998+0000] {process_utils.py:190} INFO -   |     self.__parse_headers(sodacl_dict)
[2023-10-18T17:53:11.998+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 151, in __parse_headers
[2023-10-18T17:53:11.998+0000] {process_utils.py:190} INFO -   |     self.__parse_table_checks_section(
[2023-10-18T17:53:11.999+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 215, in __parse_table_checks_section
[2023-10-18T17:53:11.999+0000] {process_utils.py:190} INFO -   |     check_str, check_configurations = self.__parse_check_configuration(check_list_element)
[2023-10-18T17:53:11.999+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 1759, in __parse_check_configuration
[2023-10-18T17:53:11.999+0000] {process_utils.py:190} INFO -   |     ignored_config_keys = [k for k in check_configurations if k != check_str]
[2023-10-18T17:53:11.999+0000] {process_utils.py:190} INFO -   | TypeError: 'NoneType' object is not iterable
[2023-10-18T17:53:11.999+0000] {process_utils.py:190} INFO - 
[2023-10-18T17:53:12.001+0000] {process_utils.py:190} INFO - [17:53:12] Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_platforms.yml
[2023-10-18T17:53:12.001+0000] {process_utils.py:190} INFO -   | 'NoneType' object is not iterable
[2023-10-18T17:53:12.001+0000] {process_utils.py:190} INFO -   | Stacktrace:
[2023-10-18T17:53:12.001+0000] {process_utils.py:190} INFO -   | Traceback (most recent call last):
[2023-10-18T17:53:12.002+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 304, in add_sodacl_yaml_file
[2023-10-18T17:53:12.002+0000] {process_utils.py:190} INFO -   |     self._parse_sodacl_yaml_str(sodacl_yaml_str=sodacl_yaml_str, file_path=file_path)
[2023-10-18T17:53:12.002+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 335, in _parse_sodacl_yaml_str
[2023-10-18T17:53:12.002+0000] {process_utils.py:190} INFO -   |     sodacl_parser.parse_sodacl_yaml_str(sodacl_yaml_str)
[2023-10-18T17:53:12.002+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 115, in parse_sodacl_yaml_str
[2023-10-18T17:53:12.002+0000] {process_utils.py:190} INFO -   |     self.__parse_headers(sodacl_dict)
[2023-10-18T17:53:12.003+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 151, in __parse_headers
[2023-10-18T17:53:12.003+0000] {process_utils.py:190} INFO -   |     self.__parse_table_checks_section(
[2023-10-18T17:53:12.003+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 215, in __parse_table_checks_section
[2023-10-18T17:53:12.004+0000] {process_utils.py:190} INFO -   |     check_str, check_configurations = self.__parse_check_configuration(check_list_element)
[2023-10-18T17:53:12.004+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 1759, in __parse_check_configuration
[2023-10-18T17:53:12.004+0000] {process_utils.py:190} INFO -   |     ignored_config_keys = [k for k in check_configurations if k != check_str]
[2023-10-18T17:53:12.004+0000] {process_utils.py:190} INFO -   | TypeError: 'NoneType' object is not iterable
[2023-10-18T17:53:12.004+0000] {process_utils.py:190} INFO - 
[2023-10-18T17:53:27.996+0000] {process_utils.py:190} INFO - /tmp/venvp5oen6ll/lib/python3.8/site-packages/google/cloud/bigquery/client.py:570: UserWarning: Cannot create BigQuery Storage client, the dependency google-cloud-bigquery-storage is not installed.
[2023-10-18T17:53:27.996+0000] {process_utils.py:190} INFO -   warnings.warn(
[2023-10-18T17:53:27.996+0000] {process_utils.py:190} INFO - [17:53:27] Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_price_overview.yml
[2023-10-18T17:53:27.997+0000] {process_utils.py:190} INFO -   | 'NoneType' object is not iterable
[2023-10-18T17:53:27.997+0000] {process_utils.py:190} INFO -   | Stacktrace:
[2023-10-18T17:53:27.997+0000] {process_utils.py:190} INFO -   | Traceback (most recent call last):
[2023-10-18T17:53:27.997+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 304, in add_sodacl_yaml_file
[2023-10-18T17:53:27.997+0000] {process_utils.py:190} INFO -   |     self._parse_sodacl_yaml_str(sodacl_yaml_str=sodacl_yaml_str, file_path=file_path)
[2023-10-18T17:53:27.997+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 335, in _parse_sodacl_yaml_str
[2023-10-18T17:53:27.998+0000] {process_utils.py:190} INFO -   |     sodacl_parser.parse_sodacl_yaml_str(sodacl_yaml_str)
[2023-10-18T17:53:27.998+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 115, in parse_sodacl_yaml_str
[2023-10-18T17:53:27.998+0000] {process_utils.py:190} INFO -   |     self.__parse_headers(sodacl_dict)
[2023-10-18T17:53:27.998+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 151, in __parse_headers
[2023-10-18T17:53:27.998+0000] {process_utils.py:190} INFO -   |     self.__parse_table_checks_section(
[2023-10-18T17:53:27.998+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 215, in __parse_table_checks_section
[2023-10-18T17:53:27.999+0000] {process_utils.py:190} INFO -   |     check_str, check_configurations = self.__parse_check_configuration(check_list_element)
[2023-10-18T17:53:27.999+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 1759, in __parse_check_configuration
[2023-10-18T17:53:27.999+0000] {process_utils.py:190} INFO -   |     ignored_config_keys = [k for k in check_configurations if k != check_str]
[2023-10-18T17:53:27.999+0000] {process_utils.py:190} INFO -   | TypeError: 'NoneType' object is not iterable
[2023-10-18T17:53:27.999+0000] {process_utils.py:190} INFO - 
[2023-10-18T17:53:27.999+0000] {process_utils.py:190} INFO - [17:53:27] Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/fct_game_pricing.yml
[2023-10-18T17:53:27.999+0000] {process_utils.py:190} INFO -   | 'NoneType' object is not iterable
[2023-10-18T17:53:28.000+0000] {process_utils.py:190} INFO -   | Stacktrace:
[2023-10-18T17:53:28.000+0000] {process_utils.py:190} INFO -   | Traceback (most recent call last):
[2023-10-18T17:53:28.000+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 304, in add_sodacl_yaml_file
[2023-10-18T17:53:28.000+0000] {process_utils.py:190} INFO -   |     self._parse_sodacl_yaml_str(sodacl_yaml_str=sodacl_yaml_str, file_path=file_path)
[2023-10-18T17:53:28.000+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 335, in _parse_sodacl_yaml_str
[2023-10-18T17:53:28.000+0000] {process_utils.py:190} INFO -   |     sodacl_parser.parse_sodacl_yaml_str(sodacl_yaml_str)
[2023-10-18T17:53:28.001+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 115, in parse_sodacl_yaml_str
[2023-10-18T17:53:28.001+0000] {process_utils.py:190} INFO -   |     self.__parse_headers(sodacl_dict)
[2023-10-18T17:53:28.001+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 151, in __parse_headers
[2023-10-18T17:53:28.001+0000] {process_utils.py:190} INFO -   |     self.__parse_table_checks_section(
[2023-10-18T17:53:28.001+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 215, in __parse_table_checks_section
[2023-10-18T17:53:28.002+0000] {process_utils.py:190} INFO -   |     check_str, check_configurations = self.__parse_check_configuration(check_list_element)
[2023-10-18T17:53:28.002+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 1759, in __parse_check_configuration
[2023-10-18T17:53:28.002+0000] {process_utils.py:190} INFO -   |     ignored_config_keys = [k for k in check_configurations if k != check_str]
[2023-10-18T17:53:28.002+0000] {process_utils.py:190} INFO -   | TypeError: 'NoneType' object is not iterable
[2023-10-18T17:53:28.002+0000] {process_utils.py:190} INFO - 
[2023-10-18T17:53:28.002+0000] {process_utils.py:190} INFO - [17:53:27] Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_publishers.yml
[2023-10-18T17:53:28.002+0000] {process_utils.py:190} INFO -   | 'NoneType' object is not iterable
[2023-10-18T17:53:28.003+0000] {process_utils.py:190} INFO -   | Stacktrace:
[2023-10-18T17:53:28.003+0000] {process_utils.py:190} INFO -   | Traceback (most recent call last):
[2023-10-18T17:53:28.003+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 304, in add_sodacl_yaml_file
[2023-10-18T17:53:28.003+0000] {process_utils.py:190} INFO -   |     self._parse_sodacl_yaml_str(sodacl_yaml_str=sodacl_yaml_str, file_path=file_path)
[2023-10-18T17:53:28.004+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 335, in _parse_sodacl_yaml_str
[2023-10-18T17:53:28.004+0000] {process_utils.py:190} INFO -   |     sodacl_parser.parse_sodacl_yaml_str(sodacl_yaml_str)
[2023-10-18T17:53:28.004+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 115, in parse_sodacl_yaml_str
[2023-10-18T17:53:28.004+0000] {process_utils.py:190} INFO -   |     self.__parse_headers(sodacl_dict)
[2023-10-18T17:53:28.005+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 151, in __parse_headers
[2023-10-18T17:53:28.005+0000] {process_utils.py:190} INFO -   |     self.__parse_table_checks_section(
[2023-10-18T17:53:28.005+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 215, in __parse_table_checks_section
[2023-10-18T17:53:28.005+0000] {process_utils.py:190} INFO -   |     check_str, check_configurations = self.__parse_check_configuration(check_list_element)
[2023-10-18T17:53:28.006+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 1759, in __parse_check_configuration
[2023-10-18T17:53:28.006+0000] {process_utils.py:190} INFO -   |     ignored_config_keys = [k for k in check_configurations if k != check_str]
[2023-10-18T17:53:28.006+0000] {process_utils.py:190} INFO -   | TypeError: 'NoneType' object is not iterable
[2023-10-18T17:53:28.006+0000] {process_utils.py:190} INFO - 
[2023-10-18T17:53:28.006+0000] {process_utils.py:190} INFO - [17:53:27] Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_release.yml
[2023-10-18T17:53:28.006+0000] {process_utils.py:190} INFO -   | 'NoneType' object is not iterable
[2023-10-18T17:53:28.006+0000] {process_utils.py:190} INFO -   | Stacktrace:
[2023-10-18T17:53:28.007+0000] {process_utils.py:190} INFO -   | Traceback (most recent call last):
[2023-10-18T17:53:28.007+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 304, in add_sodacl_yaml_file
[2023-10-18T17:53:28.007+0000] {process_utils.py:190} INFO -   |     self._parse_sodacl_yaml_str(sodacl_yaml_str=sodacl_yaml_str, file_path=file_path)
[2023-10-18T17:53:28.007+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 335, in _parse_sodacl_yaml_str
[2023-10-18T17:53:28.007+0000] {process_utils.py:190} INFO -   |     sodacl_parser.parse_sodacl_yaml_str(sodacl_yaml_str)
[2023-10-18T17:53:28.007+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 115, in parse_sodacl_yaml_str
[2023-10-18T17:53:28.008+0000] {process_utils.py:190} INFO -   |     self.__parse_headers(sodacl_dict)
[2023-10-18T17:53:28.008+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 151, in __parse_headers
[2023-10-18T17:53:28.008+0000] {process_utils.py:190} INFO -   |     self.__parse_table_checks_section(
[2023-10-18T17:53:28.008+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 215, in __parse_table_checks_section
[2023-10-18T17:53:28.008+0000] {process_utils.py:190} INFO -   |     check_str, check_configurations = self.__parse_check_configuration(check_list_element)
[2023-10-18T17:53:28.008+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 1759, in __parse_check_configuration
[2023-10-18T17:53:28.009+0000] {process_utils.py:190} INFO -   |     ignored_config_keys = [k for k in check_configurations if k != check_str]
[2023-10-18T17:53:28.009+0000] {process_utils.py:190} INFO -   | TypeError: 'NoneType' object is not iterable
[2023-10-18T17:53:28.009+0000] {process_utils.py:190} INFO - 
[2023-10-18T17:53:28.009+0000] {process_utils.py:190} INFO - [17:53:27] Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_support_language.yml
[2023-10-18T17:53:28.009+0000] {process_utils.py:190} INFO -   | 'NoneType' object is not iterable
[2023-10-18T17:53:28.009+0000] {process_utils.py:190} INFO -   | Stacktrace:
[2023-10-18T17:53:28.009+0000] {process_utils.py:190} INFO -   | Traceback (most recent call last):
[2023-10-18T17:53:28.010+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 304, in add_sodacl_yaml_file
[2023-10-18T17:53:28.010+0000] {process_utils.py:190} INFO -   |     self._parse_sodacl_yaml_str(sodacl_yaml_str=sodacl_yaml_str, file_path=file_path)
[2023-10-18T17:53:28.010+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 335, in _parse_sodacl_yaml_str
[2023-10-18T17:53:28.010+0000] {process_utils.py:190} INFO -   |     sodacl_parser.parse_sodacl_yaml_str(sodacl_yaml_str)
[2023-10-18T17:53:28.010+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 115, in parse_sodacl_yaml_str
[2023-10-18T17:53:28.011+0000] {process_utils.py:190} INFO -   |     self.__parse_headers(sodacl_dict)
[2023-10-18T17:53:28.011+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 151, in __parse_headers
[2023-10-18T17:53:28.011+0000] {process_utils.py:190} INFO -   |     self.__parse_table_checks_section(
[2023-10-18T17:53:28.011+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 215, in __parse_table_checks_section
[2023-10-18T17:53:28.012+0000] {process_utils.py:190} INFO -   |     check_str, check_configurations = self.__parse_check_configuration(check_list_element)
[2023-10-18T17:53:28.012+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 1759, in __parse_check_configuration
[2023-10-18T17:53:28.012+0000] {process_utils.py:190} INFO -   |     ignored_config_keys = [k for k in check_configurations if k != check_str]
[2023-10-18T17:53:28.013+0000] {process_utils.py:190} INFO -   | TypeError: 'NoneType' object is not iterable
[2023-10-18T17:53:28.013+0000] {process_utils.py:190} INFO - 
[2023-10-18T17:53:28.013+0000] {process_utils.py:190} INFO - [17:53:27] Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_games.yml
[2023-10-18T17:53:28.014+0000] {process_utils.py:190} INFO -   | 'NoneType' object is not iterable
[2023-10-18T17:53:28.014+0000] {process_utils.py:190} INFO -   | Stacktrace:
[2023-10-18T17:53:28.014+0000] {process_utils.py:190} INFO -   | Traceback (most recent call last):
[2023-10-18T17:53:28.014+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 304, in add_sodacl_yaml_file
[2023-10-18T17:53:28.015+0000] {process_utils.py:190} INFO -   |     self._parse_sodacl_yaml_str(sodacl_yaml_str=sodacl_yaml_str, file_path=file_path)
[2023-10-18T17:53:28.015+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 335, in _parse_sodacl_yaml_str
[2023-10-18T17:53:28.015+0000] {process_utils.py:190} INFO -   |     sodacl_parser.parse_sodacl_yaml_str(sodacl_yaml_str)
[2023-10-18T17:53:28.016+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 115, in parse_sodacl_yaml_str
[2023-10-18T17:53:28.016+0000] {process_utils.py:190} INFO -   |     self.__parse_headers(sodacl_dict)
[2023-10-18T17:53:28.016+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 151, in __parse_headers
[2023-10-18T17:53:28.017+0000] {process_utils.py:190} INFO -   |     self.__parse_table_checks_section(
[2023-10-18T17:53:28.017+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 215, in __parse_table_checks_section
[2023-10-18T17:53:28.017+0000] {process_utils.py:190} INFO -   |     check_str, check_configurations = self.__parse_check_configuration(check_list_element)
[2023-10-18T17:53:28.018+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 1759, in __parse_check_configuration
[2023-10-18T17:53:28.018+0000] {process_utils.py:190} INFO -   |     ignored_config_keys = [k for k in check_configurations if k != check_str]
[2023-10-18T17:53:28.018+0000] {process_utils.py:190} INFO -   | TypeError: 'NoneType' object is not iterable
[2023-10-18T17:53:28.018+0000] {process_utils.py:190} INFO - 
[2023-10-18T17:53:28.019+0000] {process_utils.py:190} INFO - [17:53:27] Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_developers.yml
[2023-10-18T17:53:28.019+0000] {process_utils.py:190} INFO -   | 'NoneType' object is not iterable
[2023-10-18T17:53:28.019+0000] {process_utils.py:190} INFO -   | Stacktrace:
[2023-10-18T17:53:28.019+0000] {process_utils.py:190} INFO -   | Traceback (most recent call last):
[2023-10-18T17:53:28.020+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 304, in add_sodacl_yaml_file
[2023-10-18T17:53:28.020+0000] {process_utils.py:190} INFO -   |     self._parse_sodacl_yaml_str(sodacl_yaml_str=sodacl_yaml_str, file_path=file_path)
[2023-10-18T17:53:28.020+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 335, in _parse_sodacl_yaml_str
[2023-10-18T17:53:28.021+0000] {process_utils.py:190} INFO -   |     sodacl_parser.parse_sodacl_yaml_str(sodacl_yaml_str)
[2023-10-18T17:53:28.021+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 115, in parse_sodacl_yaml_str
[2023-10-18T17:53:28.021+0000] {process_utils.py:190} INFO -   |     self.__parse_headers(sodacl_dict)
[2023-10-18T17:53:28.021+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 151, in __parse_headers
[2023-10-18T17:53:28.022+0000] {process_utils.py:190} INFO -   |     self.__parse_table_checks_section(
[2023-10-18T17:53:28.022+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 215, in __parse_table_checks_section
[2023-10-18T17:53:28.022+0000] {process_utils.py:190} INFO -   |     check_str, check_configurations = self.__parse_check_configuration(check_list_element)
[2023-10-18T17:53:28.022+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 1759, in __parse_check_configuration
[2023-10-18T17:53:28.022+0000] {process_utils.py:190} INFO -   |     ignored_config_keys = [k for k in check_configurations if k != check_str]
[2023-10-18T17:53:28.022+0000] {process_utils.py:190} INFO -   | TypeError: 'NoneType' object is not iterable
[2023-10-18T17:53:28.023+0000] {process_utils.py:190} INFO - 
[2023-10-18T17:53:28.023+0000] {process_utils.py:190} INFO - [17:53:27] Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_platforms.yml
[2023-10-18T17:53:28.023+0000] {process_utils.py:190} INFO -   | 'NoneType' object is not iterable
[2023-10-18T17:53:28.023+0000] {process_utils.py:190} INFO -   | Stacktrace:
[2023-10-18T17:53:28.023+0000] {process_utils.py:190} INFO -   | Traceback (most recent call last):
[2023-10-18T17:53:28.023+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 304, in add_sodacl_yaml_file
[2023-10-18T17:53:28.023+0000] {process_utils.py:190} INFO -   |     self._parse_sodacl_yaml_str(sodacl_yaml_str=sodacl_yaml_str, file_path=file_path)
[2023-10-18T17:53:28.024+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/scan.py", line 335, in _parse_sodacl_yaml_str
[2023-10-18T17:53:28.024+0000] {process_utils.py:190} INFO -   |     sodacl_parser.parse_sodacl_yaml_str(sodacl_yaml_str)
[2023-10-18T17:53:28.024+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 115, in parse_sodacl_yaml_str
[2023-10-18T17:53:28.024+0000] {process_utils.py:190} INFO -   |     self.__parse_headers(sodacl_dict)
[2023-10-18T17:53:28.024+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 151, in __parse_headers
[2023-10-18T17:53:28.024+0000] {process_utils.py:190} INFO -   |     self.__parse_table_checks_section(
[2023-10-18T17:53:28.025+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 215, in __parse_table_checks_section
[2023-10-18T17:53:28.025+0000] {process_utils.py:190} INFO -   |     check_str, check_configurations = self.__parse_check_configuration(check_list_element)
[2023-10-18T17:53:28.025+0000] {process_utils.py:190} INFO -   |   File "/tmp/venvp5oen6ll/lib/python3.8/site-packages/soda/sodacl/sodacl_parser.py", line 1759, in __parse_check_configuration
[2023-10-18T17:53:28.025+0000] {process_utils.py:190} INFO -   |     ignored_config_keys = [k for k in check_configurations if k != check_str]
[2023-10-18T17:53:28.025+0000] {process_utils.py:190} INFO -   | TypeError: 'NoneType' object is not iterable
[2023-10-18T17:53:28.025+0000] {process_utils.py:190} INFO - 
[2023-10-18T17:53:29.590+0000] {process_utils.py:190} INFO - Running Soda Scan...
[2023-10-18T17:53:29.590+0000] {process_utils.py:190} INFO - INFO   | Soda Core 3.0.51
[2023-10-18T17:53:29.591+0000] {process_utils.py:190} INFO - DEBUG  | Reading configuration file "/opt/***/includes/soda_quality/configuration.yml"
[2023-10-18T17:53:29.591+0000] {process_utils.py:190} INFO - DEBUG  | Reading SodaCL file "/opt/***/includes/soda_quality/checks/transform/dim_price_overview.yml"
[2023-10-18T17:53:29.591+0000] {process_utils.py:190} INFO - ERROR  | Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_price_overview.yml | 'NoneType' object is not iterable
[2023-10-18T17:53:29.591+0000] {process_utils.py:190} INFO - DEBUG  | Reading SodaCL file "/opt/***/includes/soda_quality/checks/transform/fct_game_pricing.yml"
[2023-10-18T17:53:29.594+0000] {process_utils.py:190} INFO - ERROR  | Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/fct_game_pricing.yml | 'NoneType' object is not iterable
[2023-10-18T17:53:29.594+0000] {process_utils.py:190} INFO - DEBUG  | Reading SodaCL file "/opt/***/includes/soda_quality/checks/transform/dim_publishers.yml"
[2023-10-18T17:53:29.595+0000] {process_utils.py:190} INFO - ERROR  | Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_publishers.yml | 'NoneType' object is not iterable
[2023-10-18T17:53:29.595+0000] {process_utils.py:190} INFO - DEBUG  | Reading SodaCL file "/opt/***/includes/soda_quality/checks/transform/dim_release.yml"
[2023-10-18T17:53:29.595+0000] {process_utils.py:190} INFO - ERROR  | Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_release.yml | 'NoneType' object is not iterable
[2023-10-18T17:53:29.595+0000] {process_utils.py:190} INFO - DEBUG  | Reading SodaCL file "/opt/***/includes/soda_quality/checks/transform/dim_support_language.yml"
[2023-10-18T17:53:29.596+0000] {process_utils.py:190} INFO - ERROR  | Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_support_language.yml | 'NoneType' object is not iterable
[2023-10-18T17:53:29.596+0000] {process_utils.py:190} INFO - DEBUG  | Reading SodaCL file "/opt/***/includes/soda_quality/checks/transform/dim_games.yml"
[2023-10-18T17:53:29.596+0000] {process_utils.py:190} INFO - ERROR  | Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_games.yml | 'NoneType' object is not iterable
[2023-10-18T17:53:29.596+0000] {process_utils.py:190} INFO - DEBUG  | Reading SodaCL file "/opt/***/includes/soda_quality/checks/transform/dim_developers.yml"
[2023-10-18T17:53:29.596+0000] {process_utils.py:190} INFO - ERROR  | Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_developers.yml | 'NoneType' object is not iterable
[2023-10-18T17:53:29.597+0000] {process_utils.py:190} INFO - DEBUG  | Reading SodaCL file "/opt/***/includes/soda_quality/checks/transform/dim_platforms.yml"
[2023-10-18T17:53:29.597+0000] {process_utils.py:190} INFO - ERROR  | Could not add SodaCL file /opt/***/includes/soda_quality/checks/transform/dim_platforms.yml | 'NoneType' object is not iterable
[2023-10-18T17:53:29.597+0000] {process_utils.py:190} INFO - DEBUG  | Scan execution starts
[2023-10-18T17:53:29.597+0000] {process_utils.py:190} INFO - DEBUG  | Query games.dim_price_overview.schema[dim_price_overview]:
[2023-10-18T17:53:29.597+0000] {process_utils.py:190} INFO - SELECT column_name, data_type, is_nullable FROM INFORMATION_SCHEMA.COLUMNS WHERE table_name = 'dim_price_overview';
[2023-10-18T17:53:29.598+0000] {process_utils.py:190} INFO - DEBUG  | Query games.fct_game_pricing.schema[fct_game_pricing]:
[2023-10-18T17:53:29.598+0000] {process_utils.py:190} INFO - SELECT column_name, data_type, is_nullable FROM INFORMATION_SCHEMA.COLUMNS WHERE table_name = 'fct_game_pricing';
[2023-10-18T17:53:29.598+0000] {process_utils.py:190} INFO - DEBUG  | Query games.dim_publishers.schema[dim_publishers]:
[2023-10-18T17:53:29.598+0000] {process_utils.py:190} INFO - SELECT column_name, data_type, is_nullable FROM INFORMATION_SCHEMA.COLUMNS WHERE table_name = 'dim_publishers';
[2023-10-18T17:53:29.598+0000] {process_utils.py:190} INFO - DEBUG  | Query games.dim_release.schema[dim_release]:
[2023-10-18T17:53:29.598+0000] {process_utils.py:190} INFO - SELECT column_name, data_type, is_nullable FROM INFORMATION_SCHEMA.COLUMNS WHERE table_name = 'dim_release';
[2023-10-18T17:53:29.598+0000] {process_utils.py:190} INFO - DEBUG  | Query games.dim_support_language.schema[dim_support_language]:
[2023-10-18T17:53:29.599+0000] {process_utils.py:190} INFO - SELECT column_name, data_type, is_nullable FROM INFORMATION_SCHEMA.COLUMNS WHERE table_name = 'dim_support_language';
[2023-10-18T17:53:29.599+0000] {process_utils.py:190} INFO - DEBUG  | Query games.dim_games.schema[dim_games]:
[2023-10-18T17:53:29.599+0000] {process_utils.py:190} INFO - SELECT column_name, data_type, is_nullable FROM INFORMATION_SCHEMA.COLUMNS WHERE table_name = 'dim_games';
[2023-10-18T17:53:29.599+0000] {process_utils.py:190} INFO - DEBUG  | Query games.dim_developers.schema[dim_developers]:
[2023-10-18T17:53:29.599+0000] {process_utils.py:190} INFO - SELECT column_name, data_type, is_nullable FROM INFORMATION_SCHEMA.COLUMNS WHERE table_name = 'dim_developers';
[2023-10-18T17:53:29.599+0000] {process_utils.py:190} INFO - DEBUG  | Query games.dim_platforms.schema[dim_platforms]:
[2023-10-18T17:53:29.600+0000] {process_utils.py:190} INFO - SELECT column_name, data_type, is_nullable FROM INFORMATION_SCHEMA.COLUMNS WHERE table_name = 'dim_platforms';
[2023-10-18T17:53:29.600+0000] {process_utils.py:190} INFO - INFO   | Scan summary:
[2023-10-18T17:53:29.608+0000] {process_utils.py:190} INFO - DEBUG  | 8/8 queries OK
[2023-10-18T17:53:29.609+0000] {process_utils.py:190} INFO - DEBUG  |   games.dim_price_overview.schema[dim_price_overview] [OK] 0:00:01.896192
[2023-10-18T17:53:29.609+0000] {process_utils.py:190} INFO - DEBUG  |   games.fct_game_pricing.schema[fct_game_pricing] [OK] 0:00:01.540624
[2023-10-18T17:53:29.609+0000] {process_utils.py:190} INFO - DEBUG  |   games.dim_publishers.schema[dim_publishers] [OK] 0:00:01.826228
[2023-10-18T17:53:29.609+0000] {process_utils.py:190} INFO - DEBUG  |   games.dim_release.schema[dim_release] [OK] 0:00:01.637677
[2023-10-18T17:53:29.609+0000] {process_utils.py:190} INFO - DEBUG  |   games.dim_support_language.schema[dim_support_language] [OK] 0:00:02.259733
[2023-10-18T17:53:29.610+0000] {process_utils.py:190} INFO - DEBUG  |   games.dim_games.schema[dim_games] [OK] 0:00:01.718774
[2023-10-18T17:53:29.610+0000] {process_utils.py:190} INFO - DEBUG  |   games.dim_developers.schema[dim_developers] [OK] 0:00:01.504712
[2023-10-18T17:53:29.610+0000] {process_utils.py:190} INFO - DEBUG  |   games.dim_platforms.schema[dim_platforms] [OK] 0:00:01.463240
[2023-10-18T17:53:29.610+0000] {process_utils.py:190} INFO - INFO   | 8/8 checks PASSED:
[2023-10-18T17:53:29.610+0000] {process_utils.py:190} INFO - INFO   |     dim_price_overview in games
[2023-10-18T17:53:29.610+0000] {process_utils.py:190} INFO - INFO   |       Schema Check [/opt/***/includes/soda_quality/checks/transform/dim_price_overview.yml] [PASSED]
[2023-10-18T17:53:29.611+0000] {process_utils.py:190} INFO - INFO   |         schema_measured = [app_id INT64, initial_price INT64, final_price INT64, discount_pct INT64, discount_amount FLOAT64, formatted_initial_price STRING, formatted_final_price STRING, price_with_tax FLOAT64, formatted_price_with_tax STRING]
[2023-10-18T17:53:29.611+0000] {process_utils.py:190} INFO - INFO   |     fct_game_pricing in games
[2023-10-18T17:53:29.611+0000] {process_utils.py:190} INFO - INFO   |       Schema Check [/opt/***/includes/soda_quality/checks/transform/fct_game_pricing.yml] [PASSED]
[2023-10-18T17:53:29.612+0000] {process_utils.py:190} INFO - INFO   |         schema_measured = [app_id INT64, formatted_initial_price STRING, total_sales FLOAT64, formatted_price_with_tax STRING, total_discounts FLOAT64, price_with_tax FLOAT64, num_sales INT64, max_discount INT64, min_price INT64, max_price INT64, total_sales_with_tax FLOAT64, total_sales_with_tax_discounted INT64, game_name STRING, game_description STRING, game_is_free INT64, fullgame_app_id INT64, fullgame_name STRING, type STRING, genre_id STRING, type_genre STRING]
[2023-10-18T17:53:29.612+0000] {process_utils.py:190} INFO - INFO   |     dim_publishers in games
[2023-10-18T17:53:29.612+0000] {process_utils.py:190} INFO - INFO   |       Schema Check [/opt/***/includes/soda_quality/checks/transform/dim_publishers.yml] [PASSED]
[2023-10-18T17:53:29.612+0000] {process_utils.py:190} INFO - INFO   |         schema_measured = [publisher_id INT64, publishers_name STRING]
[2023-10-18T17:53:29.612+0000] {process_utils.py:190} INFO - INFO   |     dim_release in games
[2023-10-18T17:53:29.613+0000] {process_utils.py:190} INFO - INFO   |       Schema Check [/opt/***/includes/soda_quality/checks/transform/dim_release.yml] [PASSED]
[2023-10-18T17:53:29.613+0000] {process_utils.py:190} INFO - INFO   |         schema_measured = [release_id INT64, release_date STRING, coming_soon INT64]
[2023-10-18T17:53:29.613+0000] {process_utils.py:190} INFO - INFO   |     dim_support_language in games
[2023-10-18T17:53:29.613+0000] {process_utils.py:190} INFO - INFO   |       Schema Check [/opt/***/includes/soda_quality/checks/transform/dim_support_language.yml] [PASSED]
[2023-10-18T17:53:29.613+0000] {process_utils.py:190} INFO - INFO   |         schema_measured = [language_id INT64, interface_languages STRING, subtitles_languages STRING, fullaudio_languages STRING]
[2023-10-18T17:53:29.613+0000] {process_utils.py:190} INFO - INFO   |     dim_games in games
[2023-10-18T17:53:29.614+0000] {process_utils.py:190} INFO - INFO   |       Schema Check [/opt/***/includes/soda_quality/checks/transform/dim_games.yml] [PASSED]
[2023-10-18T17:53:29.614+0000] {process_utils.py:190} INFO - INFO   |         schema_measured = [app_id INT64, name STRING, description STRING, fullgame_app_id INT64, fullgame_name STRING, type STRING, is_free INT64, genre_id STRING, type_genre STRING]
[2023-10-18T17:53:29.614+0000] {process_utils.py:190} INFO - INFO   |     dim_developers in games
[2023-10-18T17:53:29.614+0000] {process_utils.py:190} INFO - INFO   |       Schema Check [/opt/***/includes/soda_quality/checks/transform/dim_developers.yml] [PASSED]
[2023-10-18T17:53:29.614+0000] {process_utils.py:190} INFO - INFO   |         schema_measured = [developer_id INT64, developers_name STRING]
[2023-10-18T17:53:29.614+0000] {process_utils.py:190} INFO - INFO   |     dim_platforms in games
[2023-10-18T17:53:29.615+0000] {process_utils.py:190} INFO - INFO   |       Schema Check [/opt/***/includes/soda_quality/checks/transform/dim_platforms.yml] [PASSED]
[2023-10-18T17:53:29.615+0000] {process_utils.py:190} INFO - INFO   |         schema_measured = [platform_id INT64, windows INT64, mac INT64, linux INT64]
[2023-10-18T17:53:29.615+0000] {process_utils.py:190} INFO - INFO   | 8 errors.
[2023-10-18T17:53:29.615+0000] {process_utils.py:190} INFO - INFO   | Oops! 8 errors. 0 failures. 0 warnings. 8 pass.
[2023-10-18T17:53:29.615+0000] {process_utils.py:190} INFO - INFO   | Sending results to Soda Cloud
[2023-10-18T17:53:29.615+0000] {process_utils.py:190} INFO - INFO   | Soda Cloud Trace: 2339130069186439979
[2023-10-18T17:53:29.615+0000] {process_utils.py:190} INFO - Soda scan successful
[2023-10-18T17:53:29.841+0000] {python.py:194} INFO - Done. Returned value was: 0
[2023-10-18T17:53:29.869+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=games_details, task_id=test_transform, execution_date=20231017T000000, start_date=20231018T175235, end_date=20231018T175329
[2023-10-18T17:53:29.893+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T17:53:29.920+0000] {taskinstance.py:2776} INFO - 3 downstream tasks scheduled from follow-on schedule check
